sdk/core/azure-core/inc/response.hpp

```diff
@@ -27,5 +27,10 @@ namespace Azure { namespace Core {
     const T& operator*() const { return this->m_value; };
 
     T ExtractValue() { return std::move(this->m_value); }
+
+    std::unique_ptr<Http::RawResponse> ExtractRawResponse()
+    {
+      return std::move(this->m_rawResponse);
```

Comment:
Remove this no matter PR https://github.com/Azure/azure-sdk-for-cpp/pull/326 is merged or rejected.

Reply:
Of course :).